.travis.yml

```diff
@@ -26,7 +27,7 @@ install:
   - python3 -m pip install --user virtualenv
   - python3 -m venv conan
   - source conan/bin/activate
-  - pip install conan
+  - pip install conan --upgrade
```

Comment:
Do you really need to upgrade conan right after to install it?

Reply:
Not really, removing it